README.md

```diff
@@ -54,16 +57,16 @@ Behind the scenes, Ansible is using `ssh -i ~/.ssh/my_ssh_key someuser@some.remo
 
 This command will make the following changes to your machine
 
-- generate trusted SSH keys in case the deployment target is the local host (required since the installer is ansible-based)
-- Pulls Quay, Redis, and Postgres containers from quay.io (if using online installer)
+- Generate trusted SSH keys, if not supplied, in case the deployment target is the local host (required since the installer is ansible-based)
+- Pulls Quay, Redis, and Postgres images from `registry.redhat.io` (if using online installer)
```

Comment:
Why not quay.io?

Reply:
Mostly for consistency. We're using registry.redhat.io in the offline bundle so to provide an identical experience the online images will also use registry.redhat.io. Happy to revisit this in the future.

Reply:
aye makes sense